libs/components/datetime/src/lib/modules/datepicker/calendar/datepicker-date.ts

```diff
@@ -1,4 +1,6 @@
 /**
+ * Represents the selected date in the calendar picker.
+ * TODO: Consider renaming this interface to better fit its purpose.
  * @internal
```

Comment:
if this is internal and so not part of our contract with consumers why not change it now?

Reply:
Done!